edm4hep.yaml

```diff
@@ -270,290 +270,290 @@ datatypes:
+      - int32_t   algorithmType  // type of the algorithm/module that created this hypothesis.
+      - float likelihood     // likelihood of this hypothesis - in a user defined normalization.
     VectorMembers:
-      - float parameters     //parameters associated with this hypothesis. Check/set collection parameters ParameterNames_PIDAlgorithmTypeName for decoding the indices.
+      - float parameters     // parameters associated with this hypothesis.
 
 
  #------ Cluster
   edm4hep::Cluster:
     Description: "Calorimeter Hit Cluster"
-    Author: "F.Gaede, DESY"
+    Author: "EDM4hep authors"
     Members:
-      - int32_t                  type            //flagword that defines the type of cluster. Bits 16-31 are used internally.
-      - float                energy          //energy of the cluster [GeV]
-      - float                energyError     //error on the energy
-      - edm4hep::Vector3f    position        //position of the cluster [mm]
-      - std::array<float,6>  positionError   //covariance matrix of the position (6 Parameters)
-      - float                iTheta          //intrinsic direction of cluster at position  Theta. Not to be confused with direction cluster is seen from IP.
-      - float                phi             //intrinsic direction of cluster at position - Phi. Not to be confused with direction cluster is seen from IP.
-      - edm4hep::Vector3f    directionError  //covariance matrix of the direction (3 Parameters) [mm^2]
+      - int32_t                  type            // flagword that defines the type of cluster. Bits 16-31 are used internally.
+      - float                energy          // energy of the cluster [GeV].
+      - float                energyError     // error on the energy.
+      - edm4hep::Vector3f    position        // position of the cluster [mm].
+      - std::array<float,6>  positionError   // covariance matrix of the position (6 Parameters)
+      - float                iTheta          // intrinsic direction of cluster at position  Theta. Not to be confused with direction cluster is seen from IP.
+      - float                phi             // intrinsic direction of cluster at position - Phi. Not to be confused with direction cluster is seen from IP.
+      - edm4hep::Vector3f    directionError  // covariance matrix of the direction (3 Parameters) [mm^2].
     VectorMembers:
-      - float   shapeParameters      //shape parameters - check/set collection parameter ClusterShapeParameters for size and names of parameters.
-      - float   subdetectorEnergies  //energy observed in a particular subdetector. Check/set collection parameter ClusterSubdetectorNames for decoding the indices of the array.
+      - float   shapeParameters      // shape parameters. This should be accompanied by a descriptive list of names in the shapeParameterNames collection level metadata, as a vector of strings with the same ordering.
```

Comment:
We could even add a constant into `edm4hep/Constants.h` that is just the name of this, so that there are no typos. Basically the same as we did for the `weights` vector member in `EventHeader`.